cmd/ndm_daemonset/filter/osdiskexcludefilter.go

```diff
@@ -103,7 +104,7 @@ func (odf *oSDiskExcludeFilter) Start() {
 		if devPath, err := mountPointUtil.GetDiskPath(); err != nil {
 			klog.Error(err)
 		} else {
-			odf.excludeDevPath = devPath
+			odf.excludeDevPath = append(odf.excludeDevPath, devPath)
 			return
```

Comment:
Same comment as above. we should not return here as multiple paths may be present.